src/Raven.Server/Documents/ETL/EtlProcess.cs

```diff
@@ -502,8 +455,7 @@ private bool CanContinueBatch(EtlStatsScope stats, TExtracted currentItem, int b
                 return false;
             }
 
-            if (currentItem.Type == EtlItemType.Document &&
```

Comment:
Why did you remove `currentItem.Type == EtlItemType.Document` check? We should check `Database.Configuration.Etl.MaxNumberOfExtractedDocuments` only for the docs

Reply:
we do, one line below.

Reply:
Okay. Makes sense, you don't have to check that `currentItem` is a document